docs/jax.rst

```diff
@@ -133,6 +133,74 @@ jax.Array (:code:`jax.Array`)
     make_array_from_single_device_arrays
     make_array_from_process_local_data
 
+Array properties and methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. autosummary::
+  :toctree: _autosummary
+
+    Array.addressable_data
```

Comment:
Can we skip addressable_data please? I want to remove it

Reply:
Will do